### ISSUE TITLE
Deployment pipeline

### ISSUE THREAD

**Issue by Aircopter** [microsoft/AirSim#2177]
Hello and thank you for your work.
I am very new in the robotic's world and I have some questions concerning Airsim integration on drones.
For the moment I managed to install Airsim and applied deep learning algorithms on the quadcopter in the simulator.
Now I would like to integrate these AI into a real drone, but having few knowledges in robotics I don't know exactly how to do it:

- If I buy a DJI M210 drone, add a jetson TX2  and integrate my models on it, will the drone react in the same way as in the simulator? I suppose it depends on the flight controller ? Therefore my question is: Is it possible to integrate the _simple_flight_ controller into the drone such as my drone will behave excalty as in the simulator? 

- How can I connect the drone to the simulator in order to do Hardware In the Loop ?  (considering there is no Pixhawk chip in DJI drone)

In summary, my question is what is a correct way to deploy simulated AI from AirSim simulator into real quadcopter ?

Any help is welcome :)

**Comment by adamdarko** [comment]
you might want to use rabbit mq for messaging position states but below is how i got it to work with the tello drone. if you have the money id suggest just buying a px4 drone kit and adding a tx2 or a nano too it as a companion pc. 


https://github.com/adamdarko/AirSimTello
